llvm/test/Transforms/LoopVectorize/X86/amdlibm-calls.ll

```diff
@@ -1600,4 +1653,163 @@ for.end:
   ret void
 }
 
+define void @exp10_f64(ptr nocapture %varray) {
+; CHECK-LABEL: @exp10_f64(
+; CHECK-VF2:    [[TMP5:%.*]] = call <2 x double> @amd_vrd2_exp10(<2 x double> [[TMP4:%.*]])
+; CHECK-VF4-NOT:    call <4 x double> @amd_vrd4_exp10(<4 x double> [[TMP4:%.*]])
```

Comment:
Check for @llvm.exp10.v4f64 instead (etc.)? That's what we do in the rest of the file.

Reply:
there are no such intrinsic present for exp10. I tried but it is not emitting llvm.exp10.v4f64. Instead, it generating @emp10.
![image](https://github.com/user-attachments/assets/9095375e-040a-4b26-b07a-5e174e0b2034)

Reply:
@RKSimon Please check the above comment

Reply:
Are we just missing exp10 handling in isTriviallyVectorizable?

Reply:
Yes, case for exp10 is not handled in bool llvm::isTriviallyVectorizable(Intrinsic::ID ID)